src/widget/wstarrating.h

```diff
@@ -32,7 +32,9 @@ class WStarRating : public QWidget, public WBaseWidget {
     void mouseMoveEvent(QMouseEvent *event);
     void mouseReleaseEvent(QMouseEvent *event);
     void leaveEvent(QEvent *);
-    
+    virtual bool event(QEvent* pEvent);
+    void fillDebugTooltip(QStringList* debug);
```

Comment:
C++11: "override" is enough in derived classes, you don't need to add "virtual" again. "override" is only allowed for virtual functions in derived classes. The compiler will complain if you add "override" and there is no corresponding virtual function with a matching signature in the base class.

Before switching to C++11 I added a trailing "/*override*/" comment to virtual functions in derived classes. This string can easily be replaced after switching to C++11 and you get the extra compiler check for free.